core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala

```diff
@@ -166,12 +164,6 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
     loadTestData()
   }
 
-  protected def initializeTimeZone(): Unit = {
```

Comment:
why do we remove initializeTimeZone ?

Reply:
Never mind, I just thought you move to another file.

Reply:
added in SharedSQLContext